integration_tests/pages/referral/on-behalf-of.ts

```diff
@@ -0,0 +1,9 @@
+import Page, { PageElement } from '../page'
+
+export default class OnBehalfOfPage extends Page {
+  constructor() {
+    super('Referrals (On Behalf Of) asdadasdwq is under construction...')
+  }
+
+  miniProfileHyperlink = (name: string): PageElement => cy.findByRole('link', { name: new RegExp(name, 'i') })
```

Comment:
I think this should be in the base `Page` as the mini profile is likely to be on the majority of pages